apps/microfrontend-platform-testing-app/src/app/messaging/publish-message/publish-message.component.html

```diff
@@ -53,10 +53,10 @@
 
 <ng-template #intent_template>
   <sci-form-field label="Intent type *">
-    <input [formControl]="form.get(DESTINATION).get(TYPE)" class="e2e-intent-type">
+    <input [formControl]="$any(form.get(DESTINATION).get(TYPE))" class="e2e-intent-type">
```

Comment:
Consider binding to form controls using `formControlName` directive to omit the typecast, except for bindings to `sci-params-enter` which unfortunately does not implement `ControlValueAccessor` yet.

Reply:
done